sentry-android-core/src/main/java/io/sentry/android/core/ManifestMetadataReader.java

```diff
@@ -333,6 +338,23 @@ private static boolean readBool(
     return value;
   }
 
+  @SuppressWarnings("deprecation")
+  private static @Nullable Boolean readBoolNullable(
```

Comment:
Alternative could be to read `metadata.getBoolean()` twice with different defaults and check if the result differs, then return `null`. Lmk if you prefer that or have a better suggestion.

Reply:
You mean keep as is or the read twice variant?

Reply:
keep as is